adage-front/src/app/components/OffersSearch/Offers/Offers.tsx

```diff
@@ -59,15 +61,23 @@ export const OffersComponent = ({
 
   return (
     <ul className="offers">
-      {offers.map((offer, index) => (
+      {offers.map((offer) => (
         <div key={offer.id}>
-          <Offer
-            canPrebookOffers={userRole == Role.redactor}
-            key={offer.id}
-            offer={offer}
-            thumbUrl={offersThumbById[offer.id]}
-          />
-          {index < offers.length - 1 && <hr className="separator" />}
+          {
+            isNewOfferDesignEnabled() ?
+              <OfferNewDesign
+                offer={offer}
+                canPrebookOffers={userRole == Role.redactor}
+                key={offer.id}
+              />
+            :
+              <Offer
+                canPrebookOffers={userRole == Role.redactor}
+                key={offer.id}
+                offer={offer}
+                thumbUrl={offersThumbById[offer.id]}
+              />
+            }
```

Comment:
c'est préférable de renommer l'ancien quand on fait du feature flag, c'est plus simple de supprimer le code quand le temps est venu 
```suggestion
            isNewOfferDesignEnabled() ?
              <Offer
                offer={offer}
                canPrebookOffers={userRole == Role.redactor}
                key={offer.id}
              />
            :
              <OfferLegacy
                canPrebookOffers={userRole == Role.redactor}
                key={offer.id}
                offer={offer}
                thumbUrl={offersThumbById[offer.id]}
              />
            }
```